agriculture/data/ir_ui_view.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="account_move_form_view" model="ir.ui.view">
+        <field name="name">account.move.form.customization</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="mode">extension</field>
+        <field name="model">account.move</field>
+        <field name="priority">160</field>
+        <field name="type">form</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='account_id']" position="after">
+                <field name="x_product_story" widget="image"/>
+            </xpath>
+        </field>
+    </record>
+    <record id="crm_lead_form_view" model="ir.ui.view">
+        <field name="name">crm.lead.form.customization</field>
+        <field name="inherit_id" ref="crm.crm_lead_view_form"/>
+        <field name="mode">extension</field>
+        <field name="model">crm.lead</field>
+        <field name="priority">160</field>
+        <field name="type">form</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='stage_id']" position="after">
+                <button name="%(action_lead_survey_start)d" string="Survey Start" class="btn btn-primary" type="action" invisible="(x_answer != False) or (x_survey == False)"/>
+            </xpath>
+            <xpath expr="//field[@name='tag_ids']" position="after">
+                <field name="x_survey"/>
+                <field name="x_answer" invisible="x_answer == False" readonly="1" options="{'no_create':true,'no_quick_create':true,'no_create_edit':true}"/>
+            </xpath>
+        </field>
+    </record>
+    <record id="product_product_form_view" model="ir.ui.view">
+        <field name="name">product.template.product.form.customization</field>
+        <field name="inherit_id" ref="product.product_template_only_form_view"/>
+        <field name="mode">extension</field>
+        <field name="model">product.template</field>
+        <field name="priority">160</field>
+        <field name="type">form</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='uom_po_id']" position="after">
+                <field name="x_binary_field_product_template"/>
```

Comment:
You probably need a widget here, to see the image

Reply:
Thanks. I wonder if that is the best place though. I'll discuss that with PO.

Reply:
ok

Reply:
We keep it here, but resize it a bit because it is too large. Something like `options="{'size': [0,90]}`